### PR TITLE
Wrapped libtiff5 and libltdl7

### PR DESCRIPTION
**Pull request by kaixindeken**
* wrapped libtiff5

* wrapped libltdl7

### CODE REVIEW

**Comment by kaixindeken** [comment]
Need help with github action error below @ptitSeb

**Comment by kaixindeken** [comment]
Still in Progress

**Comment by kaixindeken** [comment]
Done

**Comment by kaixindeken** [comment]
Looking good while rendering the PDF version of box86.com

**Comment by ptitSeb** [comment]
On what architecture is that screenshot? Arm? RV64 or LA64 (or something else)?

File: src/wrapped/wrappedlibtiff5_private.h [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+//GO(TIFFInitCCITTFax4
+GO(TIFFInitJBIG, iFpi)
+GO(TIFFInitZIP, iFpi)
+//GO(TIFFInitPixarLog
+//GO(TIFFInitSGILog
+GO(TIFFInitLZMA, iFpi)
+GO(TIFFInitZSTD, iFpi)
+GO(TIFFInitWebP, iFpi)
+// GO(TIFFCIELabToXYZ, vFpuiippp)
+GO(TIFFXYZToRGB, iFpppp)
```
Comment (ksco) [inline src/wrapped/wrappedlibtiff5_private.h:32]: void TIFFXYZToRGB(TIFFCIELabToRGB *cielab, float X, float Y, float Z, uint32_t *R, uint32_t *G, uint32_t *B)

I’m on my phone, so I randomly picked a function to check, and it seems badly wrong?

Comment (ptitSeb) [reply src/wrapped/wrappedlibtiff5_private.h:32]: Ok, I concur, this function has a bad signature.

@kaixindeken how are you making the function signatures? Should I revert all your PR?

Comment (ptitSeb) [reply src/wrapped/wrappedlibtiff5_private.h:32]: (man page from here: https://www.manpagez.com/man/3/TIFFXYZToRGB/ )

**Comment by ksco** [comment]
If you don’t understand how libwrapping works, feel free to talk to me first instead of submit something that is very wrong.

**Comment by kaixindeken** [comment]
> If you don’t understand how libwrapping works, feel free to talk to me first instead of submit something that is very wrong.

my apologise, please feel free to revert my pr. @ksco @ptitSeb

File: src/wrapped/wrappedlibtiff5_private.h (hunk 1 above)
Comment (kaixindeken) [reply src/wrapped/wrappedlibtiff5_private.h:32]: my apologis, i'll fix it right now. please do not revert this pr. i'll create a new pr to fix it all.

**Pull request merged**